contents/docs/api/feature-flags.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Feature Flags
+sidebar: Docs
+showTitle: true
+---
+
+<span class='note-block'>For instructions on how to authenticate to use this endpoint, see [API Overview](/docs/api/api).</span><br />
+
+PostHog provides you with an API endpoint to create and update your [feature flags](/docs/features/feature-flags). This section refers to that endpoint specifically. 
+
+If you're looking to use feature flags on your application, you can either use our [JavaScript Integration](/docs/integrations/js-integration#feature-flags) or our [dedicated endpoint](/docs/api/post-only-endpoints#feature-flags) for checking if feature flags are enabled for a given user.
+
+## Create Feature Flags
+
+```shell
+POST /feature_flag/
+```
+
+### Data
+
+<span class="table-borders">
+
+| Attribute | Type | Required | Default | Description |
+| --- | --- | --- | ------------------------- | ------------------------------------------------------- |
+| `name` | String | Yes | | The name you wish to give to your feature flag. Helps identify the flag easily when using the PostHog UI. |
+| `key` | String | Yes | | The actual identifier of your flag to be used when referencing it in your app's logic. |
+| `active` | Boolean | No | `False` | Toggles the flag on and off. |
+| `rollout_percentage` | Integer | No | `None` | Percentage of users with the specified filters that ther given flag will apply to. If filters are not specified, this will be a percentage of your total users.  |
+| `filters` | Hash Table | No | `{}` | Properties of users to be matched for a flag to be on. |
+| `personal_api_key` | String | Yes | | Your Personal API Key from /setup. |
+
+
+</span>
+
+### Example Request
+
+```bash
+curl -i -X POST \
+-F 'active=true' \
+-F 'filter={properties: [{key: "signed_up", value: "true", type: "person"}]}' \
+-F 'name=Feature Flag' \
+-F 'key=flag' \
+-F 'rollout_percentage=20' \
+-F 'personal_api_key=<your_key>' \
+-L -v 'https://[your-instance].com/api/feature_flag/'
+```
+
+### Example Response
+
+```bash
+{
+    "id": 123,
+    "name": "My Flag",
+    "key": "my-flag",
+    "rollout_percentage": 20,
+    "filters": {
+        "properties": [
+            {
+                "key": "signed_up",
+                "type": "person",
+                "value": "true"
+            }
+        ]
+    },
+    "deleted": false,
+    "active": true,
+    "created_by": {
+        "id": 123,
+        "distinct_id": "4aa16b878667376032ced03c6f434b5932f",
+        "first_name": "John",
+        "email": "john.smith@fakeemail.com"
+    },
+    "created_at": "2020-09-02T18:36:15.993777Z"
+}
+```
+
+## Get the Details of a Flag
+
+```shell
+GET /feature_flag/123/
+```
+
+### Data
+
+To get the details of a given feature flag, pass its ID directly on the path. You can pass your Personal API Key as an HTTP header, and no additional data is required.
+
+### Example Request
+
+```bash
+curl -i \
+-H "Authorization: <your_key> Bearer" \
```

Comment:
Keyword before token:
`Authorization: Bearer $YOUR_KEY`

Reply:
For consistency, I can switch it around no problem, this also works though

Reply:
That's weird, it should only work with keyword before, that's how the regex is done and also how it works for me. 🤔